README.md

```diff
@@ -1 +1,63 @@
-# git-flow
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+Leaderboard
+To-do list" is a tool that helps to displays leaderboard score. It also adds the new score to the board
+
+## Built With
+
+html
+css
+Javascript
+Es6 webpack
+modules
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+es6 webpack
+understanding of Modules
+
+### Setup
+
+webpack setup
+
+### Install
+
+### Usage
+
```

Comment:
- [ ] For subsequent projects that build on this one, the instruction for setting up and running the project would be the same so we can add them now. Kindly include detailed instructions on how to set up and run the app locally, i.e. from cloning to starting the app `(npm start)`

Reply:
Done as advised